app/controllers/admin_controller.rb

```diff
@@ -36,6 +36,7 @@ def user_feedback
     @feedback = @feedback.order('feedbacks.id DESC').paginate(page: params[:page], per_page: 100)
     @feedback_count = @feedback.count
     @invalid_count = @feedback.where(is_invalidated: true).count
+    @feedback_count_today = @feedback.where('feedbacks.updated_at > ?', DateTime.now.beginning_of_day).count
```

Comment:
Ah, I figured I might need the time in there too but didn't test it without to be fair..